apps/passport-server/migrations/46_topic_pkey.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE telegram_chat_topics DROP CONSTRAINT telegram_chat_anon_topics_pkey;
+ALTER TABLE telegram_chat_topics ADD CONSTRAINT telegram_chat_topics_idx UNIQUE (telegram_chat_id, topic_id);
```

Comment:
can we add a comment to make it more clear what we're doing here? something like 
```
-- Converting primary key constraint to a unique index as we're adding a new primary key `id` column
```